data/css/modules/card/_media.scss

```diff
@@ -22,7 +22,6 @@
         font-size: .62em;  /* 10pt * 1/12 * 0.75 DPI correction factor */
         font-family: $display-font;
         color: $accent-light-color;
-        transition: color 250ms ease-in-out;
```

Comment:
The commit description mentions changes in the sidebar and text styling, but the commit contains many changes that are not related to that. Including changes to the non-GTK styling sheets.

Can you double check whether these changes belong here? and if they do, elaborate a bit more in the description to explain what this commit achieves.